src/Microsoft.Azure.ServiceBus/Amqp/AmqpResponseMessage.cs

```diff
@@ -79,5 +79,12 @@ public IEnumerable<TValue> GetListValue<TValue>(MapKey key)
 
             return list.Cast<TValue>();
         }
+
+        public AmqpSymbol GetResponseErrorCondition()
```

Comment:
I can't seem to find where this method is used in PR. At the same time, [`AmqpExceptionHelper`](https://github.com/Azure/azure-service-bus-dotnet/blob/peekasync/src/Microsoft.Azure.ServiceBus/Amqp/AmqpExceptionHelper.cs#L39) has it as well.